src/styles/blocks/stars.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('../../images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  height: 12px;
+  width: 12px;
+
+  margin: 2px;
+}
+
+/* this margin is used instead `gap` property */
+.stars__star:not(:last-child) {
+  margin-right: 6px;
```

Comment:
The margin-right of 6px in `.stars__star:not(:last-child)` contradicts the 2px margin set in `.stars__star`. This inconsistency might lead to unexpected layout issues. Ensure that the margins are consistent and serve the design as expected.